DatadogSessionReplay/Tests/Processor/Flattening/NodesFlattenerTests.swift

```diff
@@ -67,66 +37,33 @@ class NodesFlattenerTests: XCTestCase {
         DDAssertReflectionEqual(flattenedNodes, [coveringNode])
     }
 
-    /*
-          R
-         / \
-        I1  V2
-       /
-      V1
-    */
-    func testFlattenNodes_withMixedVisibleAndInvisibleNodes() {
```

Comment:
Invisible nodes are no longer passed to `NodesFlattener` - so this test doesn't make sense anymore 💡